src/npip.js

```diff
@@ -63,11 +63,11 @@ const installPip = async (pkg) => {
     // run get-pip.py
     await spawnPython([tmpFile, "--user", "--quiet"], opt);
     console.log("Successfully completed pip check.");
-    fs.unlinkSync(tmpFile);
     return;
   } catch (error) {
-    fs.unlinkSync(tmpFile);
     console.error("Failed to install pip locally.\n" + JSON.stringify(error));
```

Comment:
Should this maybe be `throw` instead of `console.error`? Since if we got to this step and it failed, things are looking pretty doomed.